contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,518 @@
+---
+title: Identify
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+
+---
+
+Identify enables you to to track your users across different devices, platforms (frontend and backend), and connect events from before and after they log in for the first time.
+
+Identifying users is done using the `identify` method, together with a unique ID.
+
+<MultiLanguage selector="tabs">
+
+```node
+client.identify({
+    distinctId: 'distinct_id',
+    properties: {
+        name: 'Max Hedgehog',
+        email: 'max@hedgehogmail.com',
+    },
+})
+```
+
+```javascript
+posthog.identify(
+    'distinct_id',
+    { email: 'max@hedgehogmail.com', name: 'Max Hedgehog' } 
+    {}, 
+);
+```
+
+```python
+posthog.identify(
+    'distinct_id',
+    {
+        'name': 'Max Hedgehog',
+        'email': 'max@hedgehogmail.com'
+    }
+)
+```
+
+```php
+PostHog::identify(array(
+  'distinctId' => 'distinct_id',
+  'properties' => array(
+    'name' => 'Max Hedgehog'
+    'email' => 'max@hedgehogmail.com',
+  )
+));
+```
+
+```ruby
+posthog.identify({
+  distinct_id: "distinct_id",
+  properties: {
+    name: "Max Hedgehog",
+    email: "max@hedgehogmail.com"
+  }
+})
+```
+
+```go
+client.Enqueue(posthog.Identify{
+  DistinctId: "distinct_id",
+  Properties: posthog.NewProperties().
+    Set("name", "Max Hedgehog").
+    Set("email", "max@hedgehogmail.com"),
+})
+```
+
+```java
+posthog.identify(
+  "distinct_id",
+  new HashMap<String, Object>() {
+    {
+      put("name", "Max Hedgehog");
+      put("email", "max@hedgehogmail.com");
+    }
+  }
+);
+```
+
+</MultiLanguage>
+
+> We don't recommend using emails as distinct IDs, as people changing their email will lead to extra complexity to maintaining data integrity. It is better to use database IDs or account IDs.
+
+You should call identify as soon as you're able to. Typically, this is every time your app loads for the first time, or directly after your users log in. This ensures that events sent during your users' sessions are correctly associated with them. 
+
+You can call identify multiple times for a user throughout the same session, **as long as you continue to pass the same distinct_id**.
+
+## The difference between anonymous and identified users
+
+All events within PostHog are associated with a specific person, either an **anonymous user** (i.e., if `identify` has not been called) or an **identified user**.
+
+When a user starts browsing your website or app, they'll be automatically assigned an **anonymous ID**, which is then stored locally and allows us to track anonymous users – even across different sessions. This anonymous ID is created using the user's device ID, and typically looks like this: `17b845b08de74-033c497ed2753c-35667c03-1fa400-17b845b08dfd55`.
```

Comment:
A lot of this statement depends. If you have the JS SDK or snippet, then they are automatically assigned an anon ID, but not with backend SDKs. 

Also, there are a bunch of configuration options for initializations that means it might not always be stored locally (for example, persistence: false), and can't track across sessions.